presto-tpch/src/test/java/com/facebook/presto/tpch/TestTpchMetadata.java

```diff
@@ -359,4 +360,9 @@ private ColumnStatistics createColumnStatistics(Optional<Double> distinctValuesC
                 .setNullsFraction(zeroValue())
                 .build();
     }
+
+    private static Predicate<Map<ColumnHandle, NullableValue>> convertToPredicate(TupleDomain<ColumnHandle> tupleDomain)
+    {
+        return bindings -> tupleDomain.contains(TupleDomain.fromFixedValues(bindings));
```

Comment:
Are you sure? `TupleDomain::contains` caused the problem in #9839, see https://github.com/prestodb/presto/pull/9839/commits/d4a0f2d9f03f1024449a5319a87edef549286616#diff-ccd5d11614e67178cf1dfe85be861160L411

Reply:
Here, I am not 100% sure. But this is the method you used in #9839. I just moved it here from `TpchResultSet`.

Reply:
But there, there was also `tupleDomain = filterColumns(tupleDomain, ...)`